frontend/express_html.js

```diff
@@ -0,0 +1,11 @@
+const fs =require("fs");
+const express =require("express");
+const app=express();
+app.listen(3614);
+app.use(express.static('public'));
+app.get('/form',(req,res)=>{
+    const name=req.query.username;
+    const comment=req.query.comment;
+    fs.writeFile('/home/amey/garageScript/frontend/public/data.txt',"hello"+" "+ name+" "+ comment);
```

Comment:
Why do you have hello and space. We're eventually going to create a chatroom. Name needs to be name: comment